src/hooks/__tests__/useFullScreen.spec.tsx

```diff
@@ -0,0 +1,48 @@
+import { act, renderHook } from '@testing-library/react';
+import { useFullScreen } from '../custom-hooks';
+
+describe('useFullScreen hook', () => {
+    afterEach(() => {
+        jest.restoreAllMocks();
+    });
+
+    it('adds and removes fullscreen event listeners', () => {
```

Comment:
also here, for each test case, can you add should... and then add the rest of the description
```suggestion
    it('should add and removes fullscreen event listeners', () => {
```